src/CodiceFiscale/Calculator.php

```diff
@@ -169,12 +171,12 @@ private function calculateOmocodia($temporaryCodiceFiscale)
     /**
      * Replace a section of the omocodia.
      *
-     * @param $divider The divider.
-     * @param $temporaryCodiceFiscale The first part of the codice fiscale on which make the substitutions.
-     * @param $startingIndex The starting index.
-     * @param $endingIndex The ending index.
-     * @param $characterIndex The index to use to make the substitutions on the $temporaryCodiceFiscale.
-     * @returns Returns the temporary codice fiscale with the substitutions made.
+     * @param $divider int The divider.
+     * @param $temporaryCodiceFiscale string The first part of the codice fiscale on which make the substitutions.
+     * @param $startingIndex int The starting index.
+     * @param $endingIndex int The ending index.
+     * @param $characterIndex int The index to use to make the substitutions on the $temporaryCodiceFiscale.
+     * @return string Returns the temporary codice fiscale with the substitutions made.
```

Comment:
You could remove **Returns** here and use an uppercase **T** for the initial **the**.